Maccy/History.swift

```diff
@@ -11,6 +11,7 @@ class History {
 
   init() {
     UserDefaults.standard.register(defaults: [UserDefaults.Keys.size: UserDefaults.Values.size])
+    UserDefaults.standard.register(defaults: [UserDefaults.Keys.menuSize: UserDefaults.Values.menuSize])
```

Comment:
You should join this into a single statement:
```suggestion
    UserDefaults.standard.register(defaults: [
      UserDefaults.Keys.menuSize: UserDefaults.Values.menuSize
      UserDefaults.Keys.size: UserDefaults.Values.size,
      ])
```